tests/integrationv2/providers.py

```diff
@@ -32,6 +35,12 @@ def __init__(self, options: ProviderOptions):
         self._provider_ready_condition = threading.Condition()
         self._provider_ready = False
 
+        # Directory to run commands from
+        self.cwd = None
+
+        # Store the test name
+        self.test_name = os.environ.get('PYTEST_CURRENT_TEST').split('::')[-1].split('[')[0]
```

Comment:
Is this needed? I don't see it being used anywhere.

Reply:
Criterion uses it to properly name the test.